doc/news/_preparation_next_release.md

```diff
@@ -211,6 +211,7 @@ you up to date with the multi-language support provided by Elektra.
 ## Tools
 
 - `kdb get -v` now displays if the resulting value is a default-value defined by the metadata of the key. _(Thomas Bretterbauer)_
+- `ẁeb-ui` issue #2441 is fixed in from of a workaround
```

Comment:
Looks like a recent change to the `master` branch introduced a merge conflict. Please 

1. rebase on `master`,
2. fix the spelling mistake (`from` → `form`), and
3. add your name at the end of this news item

.

Reply:
I have made these changes and pushed them